templates/zerver/emails/missed_message.txt

```diff
@@ -1,24 +1,24 @@
 Hello {{ name }},
 
-While you were away you received {{ message_count }} new message{{ messages|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
-
+While you were away you received {{ message_count }} new{% if group_pm %} group{% elif private_message %} private{% endif %} message{{ message_count|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
 
+{% if show_message_content %}
 {% for recipient_block in messages %}
     {{ recipient_block.header.plain }}
     {% for sender_block in recipient_block.senders %}
         {% if sender_block.sender %}{{ sender_block.sender }}{% endif %}{% for message_block in sender_block.content %}
-                {{ message_block.plain }}
+        {{ message_block.plain }}
         {% endfor %}
     {% endfor %}
 {% endfor %}
-
+{% endif %}
 
 Click here to log in to Zulip and view your new messages:
 
 {{ realm_uri }}
 
 {% if reply_to_zulip  %}
```

Comment:
same comment as above; possibly we should just change the definition of reply_to_zulip in the python code that calls this template.